pysat/instruments/methods/general.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""Provides generalized routines for integrating instruments into pysat.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import pandas as pds
+
+import pysat
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def list_files(tag=None, sat_id=None, data_path=None, format_str=None,
+               supported_tags=None, fake_daily_files_from_monthly=False,
+               two_digit_year_break=None):
+    """Return a Pandas Series of every file for chosen satellite data.
+
+    This routine is intended to be used by pysat instrument modules supporting
+    a particular NASA CDAWeb dataset.
```

Comment:
Since this routine seems to be useable for many instruments, this needs to be fixed to better explain what time of instruments it supports.